hack/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Why we have such file here? We have license in root directory

Reply:
@Uburro ?